packages/angular-playground/src/app.js

```diff
@@ -0,0 +1,36 @@
+/**
+Copyright 2016 Autodesk,Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import { ButtonDirective } from '@orion-ui/angular/lib/2016-12-01';
+
+angular
+  .module('app', [])
+  .directive('orionButton', ['$log', ButtonDirective])
```

Comment:
It is not. This is the only way to make individual elements of the angular library exportable though. If you're cool with the only way to use `orion/angular` being to import the whole `orion/angular` Angular module, then I can explicitly define the names of all the Angular providers we're creating.